QT_Py_RP2040/digital_input_neopixel.py

```diff
@@ -0,0 +1,15 @@
+"""CircuitPython Digital Input example - Blinking a built-in NeoPixel LED using a button switch."""
+import board
+import digitalio
+import neopixel
+
+pixel = neopixel.NeoPixel(board.NEOPIXEL, 1)
+
+button = digitalio.DigitalInOut(board.BUTTON)
+button.switch_to_input(pull=digitalio.Pull.UP)
+
+while True:
+    if not button.value:
+        pixel.fill((255, 0, 0))
+    else:
+        pixel.fill(0)
```

Comment:
Would `(0, 0, 0)` be more straightfoward for beginners?

Reply:
@dhalbert Yes. I must have copied that from my quick example. Thank you for catching this.